features/us55-borrower-verified-landing-page.feature

```diff
@@ -0,0 +1,18 @@
+@us55
+
+Feature: Check content on Borrower Landing Page
+
+Acceptance Criteria
+    Page content matches wireframe
+
+Scenario Outline: Borrower arrives landing Page
+    Given I reach the Borrower Identity Verified Page
+    Then I should see <Link_button>
+    And the following <Texts>
+
+    Examples:
+      | Link_button            | Texts                                     |
```

Comment:
link_button and texts are not readable, and they are technical, the cucumber scenarios are supposed to be human readable to the likes of product owner or the BA.